### PR TITLE
Unpin `protobuf` version and update `tensorboard` version

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Follows up #13159. `tensorboard>=2.9.1` pins the version until it's compatible with the latest protobuf, so there's no need to pin it on our end. https://github.com/tensorflow/tensorboard/pull/5726#issuecomment-1150151174

### Does your PR introduce any breaking changes? If yes, please list them.
None

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [n/a] Did you make sure to **update the documentation** with your changes? (if necessary)
- [n/a] Did you write any **new necessary tests**? (not for typos and docs)
- [n/a] Did you verify new and **existing tests pass** locally with your changes?
- [x] Did you list all the **breaking changes** introduced by this pull request?
- [n/a] Did you **update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [x] Check that all items from **Before submitting** are resolved
- [x] Make sure the title is self-explanatory and the description concisely explains the PR
- [x] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @carmocca @akihironitta

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -3,9 +3,8 @@ torch>=1.8.*, <=1.11.0
 tqdm>=4.57.0, <=4.63.0
 PyYAML>=5.4, <=6.0
 fsspec[http]>=2021.05.0, !=2021.06.0, <=2022.2.0
-tensorboard>=2.2.0, <2.10.0
+tensorboard>=2.9.1, <2.10.0
```
Comment (akihironitta) [inline requirements/base.txt:6]: Only for `oldest` tests to pass in our CI, we need to update the lower bound of `tensorboard`.

**Review by carmocca — APPROVED** [review]

**Review by kaushikb11 — APPROVED** [review]

**Review by Borda — APPROVED** [review]

**Review by otaj — APPROVED** [review]

**Review by carmocca — APPROVED** [review]

**Comment by carmocca** [comment]
For this to work in our CI, we would need to increase the minimum tensorboard version
